products/knownprojects/profiles.yml

```diff
@@ -0,0 +1,14 @@
+config:
+  fail-fast: true
+
+dcp-de-postgres:
+  target: dev
+  outputs:
+    dev:
+      type: postgres
+      host: "{{ env_var('BUILD_ENGINE_HOST') }}"
+      user: "{{ env_var('BUILD_ENGINE_USER') }}"
+      password: "{{ env_var('BUILD_ENGINE_PASSWORD') }}"
+      port: "{{ env_var('BUILD_ENGINE_PORT') | as_number }}"
```

Comment:
i like newlines too, will do

and this is needed because dbt can't use the full constructed DB URL. if you meant the weird `as_number` for the port variable, without that this resolves to `port: "1234"` which fails rather than `port: 1234` ([docs](https://docs.getdbt.com/reference/dbt-jinja-functions/profiles-yml-context))